pulpcore/pulpcore/app/models/task.py

```diff
@@ -213,7 +213,7 @@ class Task(Model):
     )
 
     group = models.UUIDField(null=True)
-    state = models.TextField(choices=STATES)
+    state = models.TextField(choices=STATES, null=False)
```

Comment:
Should we default to ```waiting``` state like Pulp 2?

Reply:
I'd probably lean towards having the code explicitly set the state rather than having a default. A default could disguise a bug.